source/FreeRTOS_RA.c

```diff
@@ -270,7 +270,7 @@
 
             if( uxLast < ( uxIndex + uxLength ) )
             {
-                FreeRTOS_printf( ( "RA: Not enough bytes ( %u > %u )\n", ( unsigned ) uxIndex + uxLength, ( unsigned ) uxLast ) );
+                FreeRTOS_printf( ( "RA: Not enough bytes ( %lu > %lu )\n", uxIndex + uxLength, uxLast ) );
```

Comment:
Oops, here I prefer the version with casts:
~~~
FreeRTOS_printf( ( "RA: Not enough bytes ( %u > %u )\n", ( unsigned ) ( uxIndex + uxLength ), ( unsigned ) uxLast ) );
~~~
And mind you the extra brackets in `( uxIndex + uxLength )`.